shared/src/popup.js

```diff
@@ -77,113 +78,135 @@ function setStatus(type) {
 }
 
 async function setup() {
-  const eTokenDiv = document.querySelector('#token');
```

Comment:
I also renamed a lot of variables and functions as I was going through some other cleanup of logic.